PopPUNK/__main__.py

```diff
@@ -132,6 +132,7 @@ def get_options():
     kmerGroup.add_argument('--max-k', default = 29, type=int, help='Maximum kmer length [default = 29]')
     kmerGroup.add_argument('--k-step', default = 4, type=int, help='K-mer step size [default = 4]')
     kmerGroup.add_argument('--sketch-size', default=10000, type=int, help='Kmer sketch size [default = 10000]')
+    kmerGroup.add_argument('--reads', default = False, help='Input files are reads', action = 'store_true')
```

Comment:
Haven't seen how this is used, but I would suggest avoiding as better derived from the data. I can add as an attribute to the HDF5 database if needed

Reply:
It was already a variable in the code (and the querying functions) but was not really used for anything. At the moment I think it just switches off length-based tests - might still be useful depending on the behaviour of the length estimation from reads. Can decide once we've run the read-based evaluation?

Reply:
The length estimation from the reads is definitely less accurate (gets better the more k-mer lengths used), but should be possible to deal with using the other arguments you've added

Reply:
`--reads` gone in f96c8d1.